securityconfig/roles.yml

```diff
@@ -93,3 +93,20 @@ reports_full_access:
     - 'cluster:admin/opendistro/reports/instance/list'
     - 'cluster:admin/opendistro/reports/instance/get'
     - 'cluster:admin/opendistro/reports/menu/download'
+
```

Comment:
QQ for stored asynchronous-search results: 

Where these results are stored and who will have permission to store/write those results?

Reply:
I believe that it will be store into system index .opendistro_asynchronous_search_response 

Is there any APIs user can use to store the results? and what permission is needed to use that APIs?

Reply:
https://github.com/opendistro-for-elasticsearch/asynchronous-search/tree/master/src/main/java/com/amazon/opendistroforelasticsearch/search/asynchronous/rest - these are the APIs